docs/docs/static-query.md

```diff
@@ -42,7 +42,33 @@ export default () => (
 )
 ```
 
-Using `StaticQuery`, you can colocate a component with its data. No longer is it required to, say, pass data down from `Layout` to `Header`.
+Using `StaticQuery`, you can colocate a component with its data. No longer is it required to, say, pass data down from `Layout` to `Header`. It’s important that you write your query within the `StaticQuery` component, rather than referencing a variable from elsewhere. For example, this will result in the query “loading” for ever:
```

Comment:
> For example, this will result in the query “loading” for ever

"loading" is a bit informal. Why not write that it won't work? (Because that's the endresult)